scripts/dispatch

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -ex
+
+REPO="https://api.github.com/repos/rancher/rancherd-upgrade/dispatches"
+
+# send dispatch event to REPO
+curl -XPOST -u "${PAT_USERNAME}:${PAT_TOKEN}" \
```

Comment:
you sure we need a username? most github api stuff only requires a token, i worry we dont have the this pat_username in secrets and it is pointless to add. might want to double check you can get around this with just the token

Reply:
We use this pattern for k3s-upgrade and rke2-upgrade. So long as the secrets are setup correctly in drone we should be fine. I believe that @MonzElmasry has leveraged a past PR for another project as a template for this very thing.